helm/reana/templates/secrets.yaml

```diff
@@ -60,3 +60,18 @@ data:
   REANA_EMAIL_LOGIN: {{ .Values.notifications.email_config.login | default "team@reana.io" | b64enc }}
   REANA_EMAIL_PASSWORD: {{ .Values.notifications.email_config.password | default "CHANGEME" | b64enc }}
 {{- end }}
+---
+{{- if and .Values.ingress.enabled .Values.ingress.tls.self_signed_cert }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "reana.prefix" . }}-tls-secret
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    "helm.sh/resource-policy": keep
+type: kubernetes.io/tls
+data:
+  {{- $cert := genSelfSignedCert (.Values.reana_hostname | default "localhost") nil nil 365 }}
```

Comment:
To be on the safe side, we can perhaps use 10 years instead of 1 year for self-generated certificates, just in case somebody might use it on a longer-lasting DEV server somewhere?

Reply:
I have changed it to 10 years. We should however consider that, even though this is a self-signed certificate, once the user trusts it in the browser it will remain trusted forever. This can become a security issue: if somebody gets access to the certificate it can impersonate the server for a very long time, given the very long duration of the certificate. What do you think?